src/search/queries.py

```diff
@@ -147,6 +147,50 @@ def add_aggs(self, fields: list):
         # TODO implement user defined aggregations?
 
 
+class DataServiceQuery(AbstractSearchQuery):
+    def __init__(self, search_string: str = None, aggs: list = None, filters: list = None):
+        super().__init__(search_string)
+
+        if search_string:
+            self.add_search_string(search_string)
+        else:
+            self.query = {"match_all": {}}
+
+        self.add_aggs(aggs)
+        if filters:
+            self.body["query"] = query_with_final_boost_template(must_clause=[self.query],
+                                                                 should_clause=[autorativ_dataset_query(),
```

Comment:
Tipper autorativ_dataset_query ikke har noe særlig for seg for datatjenester, så fjern den. Sjekk open_data_query, tror kanskje det samme er tilfellet med den.

Reply:
Nei, provenance har lite med datatjenester å gjøre, så autorativ er ikke nødvendig. Kanskje open_data har noe for seg senere, når vi klarer å høste det, tror ikke vi har noen bra måte å få det med i dcat-standarden atm, men tipper vi da bør lage egen open_data_dataservice_query

Reply:
be30b95